### PR TITLE
Add jQuery UI

### PR DESCRIPTION
**Pull request by mauzey1**
Add minified jQuery UI 1.13.2 with symlink.

### CODE REVIEW

File: jquery/jquery-ui.min.js [hunk 1]
```diff
@@ -0,0 +1 @@
+jquery-ui-1.13.2.min.js
```
Comment (durack1) [inline jquery/jquery-ui.min.js:1]: Mental note - latest versions https://releases.jquery.com/

**Review by durack1 — APPROVED** [review]
@mauzey1 looks good, thanks

**Comment by durack1** [comment]
@mauzey1 just a query, it seems like there are duplications in the libraries in both dataTables (https://datatables.net/download/) and jQuery (https://releases.jquery.com/). Should we separate these to migrate the dataTables content into a new ../dataTables subdir?

**Comment by mauzey1** [comment]
@durack1 I think having separate directories for jQuery and DataTables is better organization.

**Comment by durack1** [comment]
@mauzey1 agreed, I'll pull out the dataTables specific files and moved them up into their own subdir, so we'll have ../jQuery and ../dataTables at the same level

**Pull request merged**